delphi/translators/for2py/f2grfn.py

```diff
@@ -104,22 +104,23 @@ def generate_ofp_xml(preprocessed_fortran_file, ofp_file, tester_call):
 
 
 def generate_rectified_xml(ofp_xml: str, rectified_file, tester_call):
-    """This function rectifies XMl that was generated by
-    OFP. Then, it will generate an output file, but
-    also returns rectified element tree object back
-    to the caller.
-
-    Args:
-        ofp_xml (str): A string of XML that was generated by OFP.
-        rectified_file (str): A file name that rectified XML 
-        will be written to.
-        tester_call (bool): A boolean condition that will indicate
-        whether the program was invoked standalone (False) or
-        by tester scripts (True).
-
-    Returns:
-        Element Tree (ET) Object: An object of generated rectified
-        XML.
+    """
```

Comment:
The extra indentation here in the docstrings makes sphinx-apidoc complain. This was fixed in #333, but the fix doesn't seem to have propagated to this branch. Perhaps you can try doing `git cherry-pick a22e0d506d2bb51b20c027e1a7748eece9ffbf12`?

Reply:
I used the `cherry-pick` method and was indeed able to revert the commenting style to the correct solution. @adarshp please let me know if I missed one of the docstrings in the new version